github/event_types.go

```diff
@@ -106,3 +125,243 @@ type IssueCommentEvent struct {
 	Repo   *Repository `json:"repository,omitempty"`
 	Sender *User       `json:"sender,omitempty"`
 }
+
+// CommitCommentEvent represents the payload delivered by CommitCommentEvent webhook.
+//
+// GitHub docs: https://developer.github.com/v3/activity/events/types/#commitcommentevent
+type CommitCommentEvent struct {
+	Comment *RepositoryComment `json:"comment,omitempty"`
+
+	// The following fields are only populated by Webhook events.
+	Action *string     `json:"action,omitempty"`
+	Repo   *Repository `json:"repository,omitempty"`
+	Sender *User       `json:"sender,omitempty"`
+}
+
+// CreateEvent represents the payload delivered by CreateEvent webhook.
+//
+// GitHub docs: https://developer.github.com/v3/activity/events/types/#createevent
+type CreateEvent struct {
+	Ref          *string `json:"ref,omitempty"`
+	RefType      *string `json:"ref_type,omitempty"`
```

Comment:
document possible values.  Documentation style we've used for enums like this is:

```go
// Possible values are: foo, bar, baz.
```